pyccel/codegen/wrapper/c_to_python_wrapper.py

```diff
@@ -500,6 +518,221 @@ def _build_module_exec_function(self, expr):
 
         return FunctionDef(func_name, [FunctionDefArgument(module_var)], [FunctionDefResult(result_var)], body,
                 scope = func_scope, is_static=True)
+
+    def _get_class_allocator(self, class_dtype, func = None):
+        """
+        Create the allocator for the class.
+
+        Create a function which will allocate the memory for the class. This
+        is equivalent to the `__new__` function.
```

Comment:
Maybe you meant "class object" of "class instance" rather than just "class"?

Reply:
Fixed f23ff323a1faff158c75773e4bffb73e282b3cdb